addons/l10n_pe/data/account_chart_template_data.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <function model="account.chart.template" name="try_loading_for_current_company">
-            <value eval="[ref('l10n_pe.pe_chart_template')]"/>
```

Comment:
Is it necessary to change xmlid?

Reply:
I would like to simplify (as short as possible) the CoA code and call it as the peruvian standard, if you think this can bring problems, I am ok lefting it, but as this is a competely new approach, I think it is better change even the soul of the CoA ;-).

(I can bring back the old one but I think it is better if we have a new one instead).

Reply:
reverted the change of the xml_id